tests/phpunit/src/Commands/Pull/PullDatabaseCommandTest.php

```diff
@@ -116,12 +106,14 @@ public function testPullDatabaseWithMySqlImportError(): void {
     }
   }
 
-  protected function setupPullDatabase($mysql_dump_successful, $mysql_dl_successful, $mysql_drop_successful, $mysql_create_successful, $mysql_import_successful, $mock_ide_fs = FALSE): void {
+  protected function setupPullDatabase($mysql_dl_successful, $mysql_drop_successful, $mysql_create_successful, $mysql_import_successful, $mock_ide_fs = FALSE): void {
     $applications_response = $this->mockApplicationsRequest();
     $this->mockApplicationRequest();
     $environments_response = $this->mockAcsfEnvironmentsRequest($applications_response);
     $this->createMockGitConfigFile();
     $this->mockDatabasesResponse($environments_response);
+    $this->mockDatabaseBackupsResponse($environments_response, 'profserv2');
```

Comment:
Generally, shouldn't `profserv2` be added to a variable?

Reply:
Indeed it should, but I couldn't figure out where it was coming from in our mock responses. I've opened a follow-up ticket to clean up those mock responses to avoid issues like this.